public/blog/crosswalk-app-tools-alpha.md

```diff
@@ -0,0 +1,70 @@
+We are happy to announce the alpha version for Crosswalk-app-tools. This is a preview milestone of our forthcoming Node.js based tools for Android app development. Deployment in a production environment is not encouraged at this time.
+
+Crosswalk-app-tools is licensed under the Apache License Version 2.0, please refer to the file LICENSE-APACHE-V2 included in the package.
+
+### Installation
+
+Prerequisites are functional
+
+  1. Android SDK
+  2. Apache Ant
+  3. Node.js
+
+on the system.
+
+In order to get the `crosswalk-app` script available everywhere, global npm installation is required. On most Linux distributions this can be achieved by using `sudo`.
+```
+sudo npm install -g crosswalk-app-tools
+```
+
+### Usage
+
+```
+Crosswalk Application Project and Packaging Tool
+
+    crosswalk-app create <package-id>       Create project <package-id>
+                         --crosswalk=<path> Use downloaded Crosswalk
```

Comment:
These options are not shown in the "Usage" display on the version I got: 0.4.6.  Is this new?
 --croswalk=<path>
 --channel=<name>